src/test/java/com/exasol/adapter/dialects/snowflake/SnowflakeVirtualSchemaIntegrationTestSetup.java

```diff
@@ -0,0 +1,252 @@
+                    + ". Please create a Snowflake account, get the username and store it in this project in "
+                    + USERNAME_FILE + ".");
+        }
+        if (!Files.exists(Path.of(PASSWORD_FILE))) {
+            throw new IllegalStateException("Could not find " + PASSWORD_FILE
+                    + ". Please create a Snowflake account, get the password and store it in this project in "
+                    + PASSWORD_FILE + ".");
+        }
+        if (!Files.exists(Path.of(ACCOUNTNAME_FILE))) {
+            throw new IllegalStateException("Could not find " + ACCOUNTNAME_FILE
+                    + ". Please create a Snowflake account, get the accountName (part of the specific login url you get when creating the account) and store it in this project in "
+                    + ACCOUNTNAME_FILE + ".");
+        }
+
+        this.userName = Files.readString(Path.of(USERNAME_FILE)).replace("\n", "").replace("\r", "");
+        this.password = Files.readString(Path.of(PASSWORD_FILE)).replace("\n", "").replace("\r", "");
+        this.accountName = Files.readString(Path.of(ACCOUNTNAME_FILE)).replace("\n", "").replace("\r", "");
+    }
+
+    private static void uploadDriverToBucket(final ExasolContainer<? extends ExasolContainer<?>> container)
+            throws InterruptedException, TimeoutException, BucketAccessException {
+        try {
+            container.getDriverManager().install( //
+                    JdbcDriver.builder("SNOWFLAKE_JDBC_DRIVER") //
+                            .enableSecurityManager(false) //
+                            .mainClass("net.snowflake.client.jdbc.SnowflakeDriver") //
+                            .prefix("jdbc:snowflake:") //
+                            .sourceFile(JDBC_DRIVER_PATH) //
+                            .build());
+
+        } catch (final Exception exception) {
+            throw new IllegalStateException(
+                    ExaError.messageBuilder("F-VSSF-8")
+                            .message("An error occurred while uploading the jdbc driver to the bucket.")
+                            .mitigation("Make sure the {{JDBC_DRIVER_PATH}} file exists.")
+                            .parameter("JDBC_DRIVER_PATH", JDBC_DRIVER_PATH)
+                            .mitigation("You can generate it by executing the integration test with maven.").toString(),
+                    exception);
+        }
+    }
+
+    private static void uploadVsJarToBucket(final Bucket bucket) {
+        try {
+            bucket.uploadFile(PATH_TO_VIRTUAL_SCHEMAS_JAR, VIRTUAL_SCHEMAS_JAR_NAME_AND_VERSION);
+        } catch (FileNotFoundException | BucketAccessException | TimeoutException exception) {
+            throw new IllegalStateException("Failed to upload jar to bucket " + bucket, exception);
+        }
+    }
+
+    private AdapterScript createAdapterScript(final ExasolSchema schema) {
+        final String content = "%scriptclass com.exasol.adapter.RequestDispatcher;\n" //
```

Comment:
TODO: Add path to JDBC driver, as this is not included in the adapter jar anymore (changed scope to "provided" in pom file).

Reply:
I'm not sure why, but this seems to work fine for the integration tests. It's the same way here:
https://github.com/exasol/postgresql-virtual-schema/blob/ff7bfe9e0358862adc030d6cb9a6b2fc9120290c/src/test/java/com/exasol/adapter/dialects/postgresql/PostgresVirtualSchemaIntegrationTestSetup.java#L120
Any ideas as to why?

Reply:
https://github.com/exasol/oracle-virtual-schema/blob/main/src/test/java/com/exasol/adapter/dialects/oracle/OracleVirtualSchemaIntegrationTestSetup.java#L87

Reply:
As discussed, will not touch for now. It remains a 'mystery', (there's no documentation available that talks about this in details) for now and we should probably investigate when we have more time or ask around, preferably to one of the original implementers.